behat/tests/features/UserManagement/editingUserDetails.feature

```diff
@@ -0,0 +1,115 @@
+@acs
+Feature: Editing Deputy and Client details
```

Comment:
```suggestion
Feature: Editing Deputy and Client details
```

Reply:
Gah - knew I'd leave one in. 

Incidentally, I tried to set up a generic debug suite in `behat.yml` but it was a no-go due to the way we are setting up the separate fixture contexts and requiring traits. As we would need all feature contexts in the debug suite it throws an error when trying to require traits more than once.